src/lib/routes/mav-stations/index.js

```diff
@@ -0,0 +1,40 @@
+import * as helpers from '../helpers.js';
+
+/**
+ * get train stations from mav-stations API
+ *
+ * @param api mav-station API
+ * @returns list of matching stations
+ */
+const createMavStationsRoute = (api) => async (req, res, next) => {
+  const params = req.query;
+  // default result limit is 10
+  const limit = params.limit ? Number.parseInt(params.limit) : 10;
+  const searchterm = params.query ? helpers.cleanStr(params.query) : undefined;
+  const excludeCountryIso = params.excludeCountryIso
+    ? helpers.cleanStr(params.excludeCountryIso)
+    : undefined;
+
+  await api.stationList().then(function (result) {
```

Comment:
Um beim bisherigen Code Style zu bleiben, lieber:
```suggestion
  await api.stationList().then((result) => {
```

Reply:
Gefixt.